db/schema.rb

```diff
@@ -10,14 +10,16 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20200209213022) do
+ActiveRecord::Schema.define(version: 20200309204634) do
```

Comment:
I ran `rails db:setup` once I cloned the repo. I remember it showing me an error that `AddReputationToDiscordUsers` migration was pending and that I should run `rails db:migrate`. And this is the reason for this schema version to be different.

Can anyone run `rails db:migrate:status` and check what is the latest version of the file which is `up` does it matches the one in my file or the one on master?
There something fishy going on here

Reply:
@amanpatel123 Good catch. This looks like an error on my side. I think I pushed the migration files but didn't migrate before pushing. That's why the current scheme doesn't have the new columns shown below. So you're now pushing the updated, correct schema. You can keep these changes in

Reply:
Cool!